### ISSUE TITLE
Sync to drupal.org?

### ISSUE THREAD

**Issue by joshuataylor** [webflo/inline_entity_form#25]
Would be awesome to get this syncing across to Drupal.org, so we can depend on the module for contrib modules and have them included automagically.

**Comment by webflo** [comment]
Yes lets do it. I don´t have access to the inline_entity_form repo on drupal.org. But @bojanz already offered push access to me. So it should not be an issues.

Lets move back to d.o.

**Comment by slashrsm** [comment]
Great!

**Comment by bojanz** [comment]
I've now granted you (webflo and slashrsm) full access to the d.o project.
